docs/advanced/language-extensions.md

```diff
@@ -70,10 +70,14 @@ Example:
 <SideBySide>
 
 <!-- prettier-ignore -->
+
```

Comment:
I guess because of this added whiteline prettier is no longer ignoring the block, causing the formatting change?